src/pudl/transform/ferc1.py

```diff
@@ -1670,12 +2091,12 @@ def add_calculation_corrections(
                 weight=1,
             )
         )
-        # for every calc component, also make the parent-only version
-        correction_parents = correction_components.assign(
-            table_name_parent=lambda t: t.table_name,
-            xbrl_factoid_parent=lambda x: x.xbrl_factoid,
-        ).drop(columns=["table_name", "xbrl_factoid"])
-        return pd.concat([calc_components, correction_components, correction_parents])
+        # # for every calc component, also make the parent-only version
+        # correction_parents = correction_components.assign(
+        #     table_name_parent=lambda t: t.table_name,
+        #     xbrl_factoid_parent=lambda x: x.xbrl_factoid,
+        # ).drop(columns=["table_name", "xbrl_factoid"])
```

Comment:
Should this still be commented out?

Reply:
i truly don't remember why i commented these out.... but i put em back in there